cmd/kubernetes_version.go

```diff
@@ -0,0 +1,155 @@
+package cmd
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+func getKubeVersionFromRESTAPI() (string, error) {
+	k8sVersionURL := getKubernetesURL()
+	serviceaccount := "/var/run/secrets/kubernetes.io/serviceaccount"
+	cacertfile := fmt.Sprintf("%s/ca.crt", serviceaccount)
+	tokenfile := fmt.Sprintf("%s/token", serviceaccount)
```

Comment:
Can we inject these as args to `getKubeVersionFromRESTAPI()` so we are able to test this method? Currently https://github.com/aquasecurity/kube-bench/pull/518/files#diff-5c45b92badff1436a68538572a740210R22-R42 are untested.

Reply:
I didn't think I would need a test for **getKubeVersionFromRESTAPI()** as all the scenarios in it are already covered by the methods called .

Reply:
You're right. After having a second look at this, I agree with you.